### PR TITLE
Feature/vault updates

### PR DESCRIPTION
**Pull request by matthew-garrett**

### CODE REVIEW

File: packages/features/vault/src/components/how-it-works-faq.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            </BodyText>
+            <H5
+              sx={{
+                textTransform: 'uppercase',
+                fontSize: '14px',
+                color: theme.palette.typography.light,
+              }}
+            >
+              <FormattedMessage
+                defaultMessage={`Total APY = strategy APY + (strategy APY - borrow APY) * leverage`}
```
Comment (jeffywu) [inline packages/features/vault/src/components/how-it-works-faq.tsx:119]: Captialize this sentence

File: packages/features/vault/src/vault-view/vault-summary.tsx [hunk 2]
```diff
@@ -1,34 +1,73 @@
...
+import {
+  useVaultPriceExposure,
+  useVaultExistingFactors,
+  useReturnDrivers,
+  useVaultCapacity,
+  useVaultFaq,
+} from '../hooks';
 import { PRIME_CASH_VAULT_MATURITY } from '@notional-finance/util';
 
+export interface VaultsDataProps {
```
Comment (jeffywu) [inline packages/features/vault/src/vault-view/vault-summary.tsx:29]: Can you move these configs into a different file?

**Review by jeffywu — APPROVED** [review]

File: packages/features/vault/src/components/how-it-works-faq.tsx (hunk 1 above)
Comment (matthew-garrett) [reply packages/features/vault/src/components/how-it-works-faq.tsx:119]: Do you not like textTransform: 'uppercase'?

**Comment by jeffywu** [comment]
Ah didn’t see it. Sounds good

**Pull request merged**